docs/deployed-contracts/sepolia.md

```diff
@@ -100,6 +100,21 @@ and permission-based [validator set](https://github.com/eth-clients/sepolia/issu
   - L2LidoGateway
   - L2WstETHToken
 
+### Mode
+
+##### Ethereum part
+
+- L1ERC20TokenBridge: [`0xD847780c8808fc99727001C43E6C17A198b39729`](https://sepolia.etherscan.io/address/0xD847780c8808fc99727001C43E6C17A198b39729) (proxy)
+- L1ERC20TokenBridge: [`0x514Aa1d41d840af7572509D75354A8612594eB30`](https://sepolia.etherscan.io/address/0x514Aa1d41d840af7572509D75354A8612594eB30) (impl)
+
+##### Mode part
+
+- WstETH ERC20Bridged: [`0x953757Db814392642e614DaBba794Ef4381D8292`](https://sepolia.explorer.mode.network/address/0x953757Db814392642e614DaBba794Ef4381D8292) (proxy)
+- WstETH ERC20Bridged: [`0x9C7CDB9312ddA72ACc24E4A3bE5aA6afeca2AAFE`](https://sepolia.explorer.mode.network/address/0x9C7CDB9312ddA72ACc24E4A3bE5aA6afeca2AAFE) (impl)
+- L2ERC20TokenBridge: [`0x2F3a332eff5f45cEA0486464fa4080fcF52Ea876`](https://sepolia.explorer.mode.network/address/0x2F3a332eff5f45cEA0486464fa4080fcF52Ea876) (proxy)
+- L2ERC20TokenBridge: [`0x2AD850d0c6b1846F9bBD8144362A586330a96B31`](https://sepolia.explorer.mode.network/address/0x2AD850d0c6b1846F9bBD8144362A586330a96B31) (impl)
```

Comment:
should be https://sepolia.explorer.mode.network/address/0x8Cc0183D53c8fB160BFB01fe49ff3E8A9Aa0B1F6